io.github.Adda0.Stardrop.yaml

```diff
@@ -0,0 +1,68 @@
+app-id: io.github.Adda0.Stardrop
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.dotnet7
+build-options:
+    append-path: /usr/lib/sdk/dotnet7/bin
+    append-ld-library-path: /usr/lib/sdk/dotnet7/lib
+    prepend-pkg-config-path: /app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/dotnet7/lib/pkgconfig
+command: "Stardrop"
+finish-args:
+  - "--socket=x11"
+  - "--share=ipc"
+  - "--share=network"
+  - "--device=dri"
+  - "--filesystem=home"
+  - "--socket=pulseaudio"
+modules:
+  # Taken from https://github.com/flathub/com.unity.UnityHub/blob/af171c1ee22994e7ef698004bd44ba7e8bc71cf0/openssl-1.1.yaml.
+  - name: openssl-1.1
+    buildsystem: simple
+    build-commands:
+      - ./config --prefix=/app
+      - make -j $FLATPAK_BUILDER_N_JOBS
+      - make install
+    cleanup:
+      - /share/doc
+      - /share/man
+      - '*.a'
+      # Don't mask CLI tools provided by runtime - we just want the versioned shared library
+      - /bin
+      # Don't mask unversion library symlinks from library
+      - /lib/libcrypto.so
+      - /lib/libssl.so
+    sources:
+      - type: archive
+        url: https://www.openssl.org/source/openssl-1.1.1w.tar.gz
+        sha256: cf3098950cb4d853ad95c0841f1f9c6d3dc102dccfcacd521d93925208b76ac8
+        x-checker-data:
+          type: anitya
+          project-id: 20333
+          url-template: https://www.openssl.org/source/openssl-$version.tar.gz
+  - name: Stardrop
+    buildsystem: simple
+    build-commands:
+      - "install-sdk.sh"
+      - "dotnet publish -c Release --source ./nuget-sources Stardrop/Stardrop.csproj --runtime linux-x64 --self-contained true"
```

Comment:
this still looks specific to x86_64.

Reply:
You are right. I missed this one. `dotnet` should pick the correct runtime without explicitly specifying `--runtime` argument, so I just do not specify the runtime now.